api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/config/KerasPadding.kt

```diff
@@ -52,4 +66,62 @@ internal sealed class KerasPadding {
             this.padding[3] = padding[1][1]
         }
     }
+
+
+    class ZeroPadding3D : KerasPadding {
```

Comment:
Please add KDocs for this class and make it or public if it's part of API or internal if it should be available out of the library